fabric/bin/lib/common_ledger.sh

```diff
@@ -112,15 +112,32 @@ ledger_init() {
     try rm -rf ${FABRIC_STATE_DIR}/*
 
     # 2. start orderer
+    # try to create orderer-system-channel genesisblock
+    para
+    echo "[+] Creating a genesisblock for orderer-system-channel......."
+    para
+    try ${CONFIGTXGEN_CMD} -outputBlock genesisblock -profile SampleDevModeSolo -channelID orderer-system-channel
+    sleep 1
+
     ORDERER_GENERAL_GENESISPROFILE=SampleDevModeSolo ${ORDERER_CMD} 1>${ORDERER_LOG_OUT} 2>${ORDERER_LOG_ERR} &
     export ORDERER_PID=$!
     echo "${ORDERER_PID}" > ${ORDERER_PID_FILE}
     sleep 1
     kill -0 ${ORDERER_PID} || die "Orderer quit too quickly: (for log see ${ORDERER_LOG_OUT} & ${ORDERER_LOG_ERR})"
 
+
+
+
     # 3. start peer
-    LD_LIBRARY_PATH=${LD_LIBRARY_PATH:+"$LD_LIBRARY_PATH:"}${FPC_TOP_DIR}/tlcc/enclave/lib \
-		   ${PEER_CMD} node start 1>${PEER_LOG_OUT} 2>${PEER_LOG_ERR} &
+    para
```

Comment:
reporting as output that not everything is enabled is a good idea. I would though also add a TODO comment

Reply:
sure..